yapio.c

```diff
@@ -942,6 +945,15 @@ yapio_getopts(int argc, char **argv)
         case 'h':
             yapio_print_help(YAPIO_EXIT_OK);
             break;
+        case 'i':
+            if (optarg[0] == '-')
+            {
+                printf("File not specified with option -i\n");
+                yapio_print_help(YAPIO_EXIT_ERR);
+            }
+            snprintf(yapioTestFileName, PATH_MAX + 1, "%s", optarg);
```

Comment:
i would check snprintf return as you might truncate the string. Check for -1, and ask for a filename shorter than PATH_MAX

Reply:
done